docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -0,0 +1,127 @@
+# Getting Started - Snap Users
+
+## Introduction
+Just like Docker containers, the EdgeX project creates and publishes a snap for each release to the [snap store](https://snapcraft.io/edgexfoundry).  The snap currently supports running on both amd64 and arm64 platforms.
+
+## Installing snapd
```

Comment:
I don't think we need this section. Snap has been included in the last two LTS version of Ubuntu -- if they are "Snap Users" as the section is called -- they already have snapd installed.

Reply:
agreed and removed.